src/css/main.css

```diff
@@ -11,6 +11,22 @@
 
 :root {
   --font-display: 'Bebas Neue Momentum', sans-serif;
+  --breakpoint-mobile: 600px;
+  --breakpoint-tablet-landscape: 992px;
+  --breakpoint-desktop: 1200px;
```

Comment:
? These aren't used?

Reply:
Yes I found out that I can't use the vars in breakpoints - removed

https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-custom-media

This I think we can add to our vite config but it can be done later, and then we don't have to type max-width: 123px about 1000 times.

Reply:
Removed

Yep I was trying to use variables inside the breakpoints but our Vite config doesn't allow it yet.

https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-custom-media
We can add this I think later to reduce having to write `max-width: 123px` everywhere